src/NodaTime.Test/DurationTest.Construction.cs

```diff
@@ -238,6 +256,20 @@ public void FactoryMethods_OutOfRange()
             
             // No such concept as BigInteger.Min/MaxValue, so use the values we know to be just outside valid bounds.
             AssertOutOfRange(Duration.FromNanoseconds, Duration.MinNanoseconds - 1, Duration.MaxNanoseconds + 1);
+
+            void AssertLimitsInt32(Func<int, Duration> factoryMethod, int[] allCases) =>
```

Comment:
I'd hoped so - but it doesn't out you can't overload local methods :(
(I could make them non-local methods, of course.)